### PR TITLE
feat: install command for arbitrary urls

### LINKED ISSUE #14: Command to add an arbitrary URL
It should download the file, calculate the hash and an appropriate name, and make a .toml file in the mods folder accordingly. Bonus points if it can extract Forge/Fabric metadata and assign clientside/serverside accordingly!

### PR DESCRIPTION
**Pull request by Tricked-dev**
closes #14 

lmk if the command naming is right wasn't sure what to name this one it might cause confusion with modrinth/curseforge mods this way

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
+package cmd
```
Comment (comp500) [inline cmd/install.go:1]: This should be in it's own package, like the CF/MR/GH download commands; maybe with a root command named something like `url`? Then the command would be `packwiz url install`

File: cmd/install.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	Args:    cobra.ArbitraryArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		pack, err := core.LoadPack()
+
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(args) != 2 {
```
Comment (comp500) [inline cmd/install.go:29]: Use `cobra.ExactArgs` instead of manually validating here

File: cmd/install.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+			fmt.Println("Invalid usage: packwiz install [name] [url]")
+			os.Exit(1)
+		}
+		// TODO: consider using colors for these warnings but those can have issues on windows
+		if strings.HasPrefix(args[1], "https://github.com/") {
+			fmt.Println("Consider using packwiz github add", args[1], "instead")
```
Comment (comp500) [inline cmd/install.go:35]: Could prompt the user if they want to continue? Or just refuse to continue unless the user passes an argument
This should also handle www and non-www / HTTP as well as HTTPS URLs

File: cmd/install.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	"strings"
+
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var installCmd = &cobra.Command{
+	Use:     "install [name] [url]",
+	Short:   "Install a mod from a url this is different from modrinth/curseforge install",
```
Comment (comp500) [inline cmd/install.go:18]: "Add an external file from a direct download link, for sites that are not directly supported by packwiz" maybe? this is very vague and informal

**Review by comp500 — CHANGES_REQUESTED** [review]

File: cmd/install.go (hunk 3 above)
Comment (Tricked-dev) [reply cmd/install.go:35]: want me to just check if it includes github.com? also www.githubn.com redirects to github.com

Comment (comp500) [reply cmd/install.go]: I meant for all three; a user could manually type it with `www.`. I guess you could parse the URL and get the host from there, would give a nicer error when you pass an invalid URL too

Comment (Tricked-dev) [reply cmd/install.go]: Sure I'll do parsing the URL instead

**Review by comp500 — APPROVED** [review]

**Pull request merged**